data/json/requirements/toolsets.json

```diff
@@ -1,5 +1,11 @@
 [
   {
+    "id": "forging_standard",
+    "type": "requirement",
+    "//": "Forging of steel items (per steel chunk)",
+    "tools": [ [ [ "forge", 20 ], [ "char_forge", 20 ], [ "oxy_torch", 20 ] ] ]
```

Comment:
Oxy torch charges are "worth" 4x as much for forging.
Also, why start at low granularity?